This proviso is incorrect on systems where shells other than the C shell
have implemented job control.  Any ideas on how to test for this portably?
--- krb5-1.6.1/src/appl/telnet/telnet/telnet.1	2008-02-22 17:49:04.000000000 +0800
+++ krb5-1.6.1/src/appl/telnet/telnet/telnet.1	2008-02-22 17:49:40.000000000 +0800
@@ -1286,8 +1286,6 @@ commands.
 .B z
 Suspend
 .BR telnet .
-This command only works when the user's shell is
-.IR csh (1).
 .TP
 \fB\&!\fP [\fIcommand\fP]
 Execute a single command in a subshell on the local system.  If
